aexpect/remote_door.py

```diff
@@ -471,7 +471,7 @@ def get_remote_object(object_name, session=None, host="localhost", port=9090):
                 break
             time.sleep(1)
         else:
-            raise OSError("Local object sharing failed:\n%s" % output)
+            raise OSError("Local object sharing failed:\n%s" % output) from Pyro4.errors.PyroError
```

Comment:
You need to put here the actual instance and not just the class to get the full picture.

Reply:
I guess I had a mental blackout here due to the mechanic changes of all other instances, thanks for spotting! The benefits of having more than one pair of eyes on the same changes ^^